### ISSUE TITLE
Can't use SDK 34 with Google Billing 6.2.1

### ISSUE THREAD

**Issue by stephanberger4** [distriqt/ANE-GooglePlayServices#34]
Hi , 

I'm using the ANE : com.distriqt.InAppBilling

As Google requires, I've updated the manifest `<meta-data android:name="com.google.android.play.billingclient.version" android:value="5.0.0" />` to `<meta-data android:name="com.google.android.play.billingclient.version" android:value="6.2.1" />` 
It's working. BUT we have to target SDK 33 only...
              `  <uses-sdk android:minSdkVersion="19" android:targetSdkVersion="33"/>`

If we target SDK 34, the buying function is not working in the app.. 

Do I have also have to update the file com.distriqt.InAppBilling.ane ? As it been updated ?
Have you heard any feedbacks on this ? (I'm using the last AIR SDK from Harman)

Waiting for your reply, 

Regards, 

Stephan BERGER

**Comment by marchbold** [comment]
Yes please ensure you are using the latest ANE. The version tag must match the library in the ANE.

**Comment by stephanberger4** [comment]
Thanks.
Can you remind me where I can download the last version of the ANE please ?
________________________________

**Comment by marchbold** [comment]
It's in the repository : https://github.com/distriqt/ANE-InAppBilling/
